local-dev-first-start-only.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+function blue() {
+    local TEXT="$1"
+    echo -n "\e[1;34m${TEXT}\e[0m"
+}
+
+function green() {
+    local TEXT
+    if [ "$1" == "" ]; then
+        TEXT=Done
+    else
+        TEXT="$1"
+    fi
+    echo -n "\e[1;32m${TEXT}\e[0m"
+}
+
+function red() {
+    local TEXT
+    if [ "$1" == "" ]; then
+        TEXT=Failure
+    else
+        TEXT="$1"
+    fi
+    echo -n "\e[31m${TEXT}\e[0m"
+}
+
+clear
+
+echo
+echo '########################################################################################################################'
+echo -e " `red 'All containers, images and volumes will be deleted'`"
+echo '########################################################################################################################'
+echo
+echo -ne " `red 'Are you sure [y/N]?'` "; read A
+echo
+
```

Comment:
Add a line that says if you are unsure, type N. Lots of users won't have a clue.

Reply:
Updated

Reply:
@phuongdm Is this the script I need to run to host the site locally?

Reply:
yes, did you look at the documents I listed above? if not, please ask Mark O for permission, he's owner